sources/constraints.md

```diff
@@ -40,8 +40,7 @@ __参数__
 
 __参考文献__
 
-- [Dropout: A Simple Way to Prevent Neural Networks from Overfitting]
-(http://www.cs.toronto.edu/~rsalakhu/papers/srivastava14a.pdf)
+- [Dropout: A Simple Way to Prevent Neural Networks from Overfitting](http://www.cs.toronto.edu/~rsalakhu/papers/srivastava14a.pdf)
```

Comment:
![Screenshot from 2021-03-06 17-19-54](https://user-images.githubusercontent.com/32936898/110201857-342a8100-7ea0-11eb-8427-a224c38fcb39.png)

the CR which make the link ineffective is incorrect.